<a href="https://colab.research.google.com/github/LuisaMariaMontoyaConde1994/TrabajoClaseestadistica/blob/main/actividad_1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# **CHRISTIAN CAMILO BELTRAN RODRIGUEZ**

In [1]:
import numpy as np
import matplotlib.pyplot as plt
from scipy.stats import binom

En una elección, un candidato tiene una probabilidad del $60\%$ de recibir un voto en cada votación individual. Si se emiten 20 votos.


1.   ¿Cuál es la probabilidad de que al menos obtenga 12 votos?
2.   ¿Cuál es la probabilidad de que lo más obtenga 5 votos?
3.  ¿Cuál es la probabilidd de que no obtenga votos.
4.  ¿Cuál es la probabilidad de que obtenga entre 8 y 10 votos?

1. ¿Cuál es la probabilidad de que al menos obtenga 12 votos?

$$P(X\geq 12)=1-P(X< 12)$$

In [3]:
n=20
p=0.6
x=12
prob=1-binom.cdf(x-1,n,p)
print(prob)
m=prob*100
print("La probabilidad es" ,m, "%")

0.5955987253122248
La probabilidad es 59.55987253122248 %


La probabilidad de que al menos obtenga 12 votos es del $59\%$

2. ¿Cuál es la probabilidad de que lo más obtenga 5 votos?

$$P(X\leq 5)$$

In [4]:
n=20
p=0.6
x=5
prob=binom.cdf(x,n,p)
print(prob)
m=prob*100
print("La probabilidad es" ,m, "%")

0.001611524643456288
La probabilidad es 0.1611524643456288 %


 la probabilidad de que lo más obtenga 5 votos es del $0.16\%$

3. ¿Cuál es la probabilidad de que no obtenga votos?



In [16]:
n=20
p=0.6
x=0
prob=binom.cdf(x,n,p)
print(prob)
m=prob*100
print("La probabilidad es" ,m, "%")

1.0995116277760013e-08
La probabilidad es 1.0995116277760013e-06 %


 probabilidad de que no obtenga votos es de $1\%$

4. ¿Cuál es la probabilidad de que obtenga entre 8 y 10 votos?

In [8]:
n=20
p=0.6
x=10
prob=binom.cdf(x,n,p)
print(prob)
w2=prob*100
print("La probabilidad es" ,m, "%")

0.24466279668360685
La probabilidad es 24.466279668360684 %


In [14]:
n=20
p=0.6
x=8
prob=binom.cdf(x,n,p)
print(prob)
w1=prob*100
print("La probabilidad es" ,m, "%")

0.056526367034253025
La probabilidad es 24.466279668360684 %


In [15]:
w2-w1
print('la probabilidad es' ,w2-w1, '%')

la probabilidad es 18.813642964935383 %


la probabilidad de que obtenga entre 8 y 10 votos es del  $18.81\%$